beluga_ros/test/include/beluga_ros/test/point_cloud_utils.hpp

```diff
@@ -0,0 +1,436 @@
+// Copyright 2024 Ekumen, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef BELUGA_ROS_TEST_POINT_CLOUD_UTILS_HPP
+#define BELUGA_ROS_TEST_POINT_CLOUD_UTILS_HPP
+
+#include <memory>
+#include <optional>
+#include <vector>
+
+#if BELUGA_ROS_VERSION == 1
```

Comment:
Do we still support this?

Reply:
We do. I'll be addressing https://github.com/Ekumen-OS/beluga/issues/486 in a follow-up PR. Had some tests failing at me when I timed out the other day.